examples/ckks/bootstrapping/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 		valuesWant[i] = utils.RandComplex128(-1, 1)
 	}
 
-	plaintext = encoder.EncodeNew(valuesWant, params.LogSlots())
+	plaintext = encoder.EncodeNew(valuesWant, params.MaxLevel(), params.Scale(), params.LogSlots())
```

Comment:
We should take this opportunity to discuss the `Parameter.Scale` API and the `Encoder` API wrt that.